packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -139,154 +81,4 @@ export const SubscriptionItem = ({
   );
 };
 
-type CancelSubscriptionPanelProps = {
-  plan: Plan;
-  cancelSubscription: SubscriptionsProps['cancelSubscription'];
-  customerSubscription: CustomerSubscription;
-  cancelSubscriptionStatus: SelectorReturns['cancelSubscriptionStatus'];
-};
-
-const CancelSubscriptionPanel = ({
```

Comment:
:+1: Nice to see this component really slim down with the extraction of CancelSubscriptionPanel